content/style_reuse.md

```diff
@@ -13,22 +13,59 @@ product = []
     weight = 130
 +++
 
-## Content Reuse and Hugo Shortcodes
-
 Chef docs uses [shortcodes](https://gohugo.io/content-management/shortcodes/) to maintain text that appears in more than one location and must be consistent in every location.
 
-### Writing a shortcode
+## Writing a Shortcode
+
+Shortcode files are written in **Markdown** or **HTML** and are stored in the [`layouts/shortcodes`](https://github.com/chef/chef-web-docs/tree/master/themes/docs-new/layouts/shortcodes) directory in the `chef-web-docs` repo. Most shortcodes are written in Markdown but some are in HTML.
+
+## Adding a Shortcode to a Page
 
-All shortcode files are written in **Markdown** and stored in the `layouts/shortcodes` directory in the `chef-web-docs` repo.
+There are two types of shortcodes, **Markdown** and **HTML**. The type of shortcode determines how it is added to a page and how Hugo processes the text when it renders the page into HTML.
+
+{{< note >}}
+If you add a **Markdown** shortcode to a page using **HTML** shortcode delimiters, Hugo will assume that the text is already formatted in HTML and won't run the shortcode file through its Markdown processor, leaving the bare Markdown in the HTML page output.
+{{< /note >}}
 
-### Adding a Shortcode to a Page
+### Markdown Shortcodes
 
-To include a shortcode in a Markdown file, wrap the name of the shortcode file, without the file type suffix, in double curly braces and percent characters. For example, if you wanted to add the `chef.md` shortcode to a page, add the following text to the Markdown page:
+To include a Markdown shortcode in a page, wrap the name of the shortcode file, without the file type suffix, in double curly braces and percent characters. For example, if you wanted to add the [`chef.md`](https://github.com/chef/chef-web-docs/blob/master/themes/docs-new/layouts/shortcodes/chef.md) shortcode to a page, add the following text to the Markdown page:
 
 ```go
 {{%/* chef */%}}
```

Comment:
I don't understand the `/* ... */` pair in this example.

Reply:
It prevents hugo from processing it as a shortcode.